react/features/invite/functions.js

```diff
@@ -392,6 +421,16 @@ function isMaybeAPhoneNumber(text: string): boolean {
     return Boolean(digits.length);
 }
 
+/**
+ * Checks whether a string matches a sip address format.
+ *
+ * @param {string} text - The text to check.
+ * @returns {boolean} True if provided text matches a sip address format.
+ */
+function isASipAddress(text: string): boolean {
```

Comment:
We need to consider if the user pastes a full SIP URI like `sip:alice@atlanta.com` or `sips:bob@biloxi.net`. Also, in case of an IP address, there could be a port: `sip:1234@8.8.8.8:6065` do we cover these?